src/Libraries/GeometryUIWpf/PanelSurfaceBoundaryConditions.cs

```diff
@@ -0,0 +1,53 @@
+using Autodesk.DesignScript.Geometry;
+using Autodesk.DesignScript.Runtime;
+using CoreNodeModels;
+using DSCore;
+using Dynamo.Graph.Nodes;
+using ProtoCore.AST.AssociativeAST;
+using System;
+using System.Collections.Generic;
+
+namespace GeometryUIWpf
+{
+    [IsVisibleInDynamoLibrary(true)]
+    [NodeName("PanelSurfaceBoundaryConditions")]
+    [NodeCategory("Geometry.PanelSurface")]
+    [IsDesignScriptCompatible]
+    public class PanelSurfaceBoundaryConditionDropDown : EnumBase<PanelSurfaceBoundaryCondition>
```

Comment:
@aparajit-pratap any ideas how I can add a default selected item ? (or if I should) 
Should I override the constructor and set the selectedItem = 0 ?

Reply:
Yes, ` SelectedIndex = 0;` in the constructor.

Reply:
Make `PanelSurfaceBoundaryCondition.Keep` as the default (index 0)